### ISSUE TITLE
[BUG] The line number feature is broken, on IdeaPad Duet (Chrome OS with Android app)

### ISSUE THREAD

**Issue by TRENEEDANAME** [jdanielmourao/obsidian-sanctum#108]
**Describe the bug**
The line number feature is broken, on IdeaPad Duet (Chrome OS with Android app)
The line numbers are offset from the actual text, it only happens at the end of the document but can impact productivity as it makes it impossible to modify the bottom text of a document. It only happens with Sanctum, it's updated, it didn't do anything like that before.
To go to the bottom text, I need to use the right arrow key and wait for the cursor to go onto the text.

**Expected behavior**
The line number should correspond to the actual line, I should be able to modify the bottom text easily

**Screenshots**

**Version Mobile**

**Editor**
Legacy as it is the Android app

**Comment by jdanielmourao** [comment]
Hi! I'll need a bit more information about this. A screenshot and/or GIF would be helpful. If you're able to inspect the elements, even better.
Has your previous issue regarding lists been fixed?

**Comment by TRENEEDANAME** [comment]
I can do
![Screenshot 2022-01-06 20 05 57](https://user-images.githubusercontent.com/79509045/148437563-50ba8264-cf8c-4914-82ca-56d36bebd64e.png)

https://user-images.githubusercontent.com/79509045/148437770-7f925bbb-ab81-48da-a47e-46a8a0e8328b.mp4

When I write "clicking" it's because I use my down arrow key to show that this don't work

**Comment by TRENEEDANAME** [comment]
The previous issue 

> Hi! I'll need a bit more information about this. A screenshot and/or GIF would be helpful. If you're able to inspect the elements, even better. Has your previous issue regarding lists been fixed?

Yes, the list issue is now fixed

**Comment by TRENEEDANAME** [comment]
Ok, thank you

**Comment by jdanielmourao** [comment]
Have you noticed a difference with the new update?

**Comment by TRENEEDANAME** [comment]
No, same as before, the Linux version work but, the app in itself is not usable (4GB of RAM, phone hardware running a Linux VM)

**Comment by jdanielmourao** [comment]
Since I have no way of specifically targetting this issue, I've decided to close it for the time being.
